common/views/components/ContentPage/ContentPage.tsx

```diff
@@ -102,9 +102,10 @@ const ContentPage = ({
         {sectionLevelPages.includes(id) ? (
           Header
         ) : (
-          <SpacingSection>
+          // This space is coupled to the `bottom` value in PageHeader.js
```

Comment:
You could have this value exported from one of the components and imported into the other to make the coupling explicit.